glean-core/rlb/tests/simple.rs

```diff
@@ -57,7 +58,7 @@ fn simple_lifecycle() {
 
     // Create a custom configuration to use a validating uploader.
     let dir = tempfile::tempdir().unwrap();
-    let tmpname = dir.path().display().to_string();
+    let tmpname = PathBuf::from(dir.path().display().to_string());
```

Comment:
```suggestion
    let tmpname = dir.path().to_path_buf();
```

Reply:
Thank you for pointing that out (how come I didn't notice the `to_path_buf()`method?).
The requested changes had been made.